package.json

```diff
@@ -103,24 +123,8 @@
     "semantic-release": "^17.4.7",
     "typescript": "3.9.6"
   },
-  "files": [
-    "bin",
-    "dist"
-  ],
-  "nyc": {
-    "all": true,
-    "include": [
-      "dist/htmlhint.js"
-    ],
-    "exclude": [
-      "**/*.spec.js",
-      "src/**/*.ts",
-      "bin/htmlhint"
-    ],
-    "reporter": [
-      "text",
-      "lcov"
-    ],
-    "sourceMap": false
+  "engines": {
+    "node": ">=16.13.0 <= 16.99.99",
```

Comment:
I don't think this needs to get pinned to 16, or people will get warnings on install if they aren't using these versions.
Usually only need these to flag minimal versions of Node

Reply:
I removed the limit to a higher version.